src/Microsoft.DotNet.VersionTools/Util/FileUtils.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    static class FileUtils
```

Comment:
That was my mistake--dotnet style guides say to use the modifier rather than the implicit value. If you don't mind adding it to the rest of them that would be great. :smile: